### ISSUE TITLE
Build error: Cannot find module 'fuzzy'

### ISSUE THREAD

**Issue by DannyMeister** [SteamGridDB/steam-rom-manager#50]
First time building the project in Windows 10.  I get the following error:

> ERROR in [at-loader] ./src/renderer/pipes/fuzzy.pipe.ts:2:24
>     TS2307: Cannot find module 'fuzzy'.

I'm assuming a dependency isn't set up right, but I'm going ahead and posting here quickly in case I don't have time to track it down right away.

If it's relevant, here's the warnings I got in output when I previously ran `npm install`:

> npm notice created a lockfile as package-lock.json. You should commit this file.
> npm WARN worker-loader@0.8.1 requires a peer of webpack@>=0.9 <2 || ^2.1.0-beta || ^2.2.0 but none was installed.
> npm WARN optional SKIPPING OPTIONAL DEPENDENCY: 7zip-bin-linux@1.1.0 (node_modules\7zip-bin-linux):
> npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for 7zip-bin-linux@1.1.0: wanted {"os":"linux","arch"                                   :"any"} (current: {"os":"win32","arch":"x64"})
> npm WARN optional SKIPPING OPTIONAL DEPENDENCY: 7zip-bin-mac@1.0.1 (node_modules\7zip-bin-mac):
> npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for 7zip-bin-mac@1.0.1: wanted {"os":"darwin","arch":                                   "any"} (current: {"os":"win32","arch":"x64"})
> npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.1.2 (node_modules\fsevents):
> npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.1.2: wanted {"os":"darwin","arch":"any                                   "} (current: {"os":"win32","arch":"x64"})

**Comment by FrogTheFrog** [comment]
Whoops, accidentally removed dependency while switching fuzzy libraries for list parsing. Should be ok now.

**Comment by DannyMeister** [comment]
I saw that 'fuzzy' was the name of an old dependency recently removed and replaced with fuzz-aldrin.  It was still being used in a pipe translator, which was referenced by the UI for the preview component for filter searches to show fuzzy matches.

Temporarily I removed the pipe translater and its usage, but I get further build errors: "loaderUtils.getOptions is not a function".  Seems to be the recursive requires loader is chaining down into some packages with depend on an older version of loader-utils.  Version 1.0.0 had a breaking change which requires the use of getOptions().
